clar2wasm/src/wasm_generator.rs

```diff
@@ -21,6 +21,9 @@ use walrus::{
     ValType,
 };
 
+/// First free position after data directly defined in standard.wat
+pub const END_OF_STANDARD_DATA: u32 = 288;
```

Comment:
I don't love this being hard-coded here. We should instead be able to get this size after we load the standard lib.

Reply:
I agree, I wanted to postpone that for Future-Acaccia :/
I still need to increment this value in my coming PR for `hash160`, do you mind if I figure that out later?

Reply:
Sure, I'm okay with that.